### ISSUE TITLE
add DefinedTerm as expected type to learningResourceType

### ISSUE THREAD

**Issue by sroertgen** [schemaorg/schemaorg#2764]
Hello 👋
is it possible to add `DefinedTerm` as an expected type to <http://schema.org/learningResourceType>?

Background is this:
My project is working as a metadata-hub for educational resources in Germany. We gather material from many different sources and some of them use different vocabularies for defining their learning resource types. We connect the different content sources in our hub, so they can interchange their material. 

Problem is:
They will only accept material with metadata they can understand. Thus we are thinking of drawing relations between the vocabs in the background and give each player what they understand. Therefore we are handling `learningResourceType` entries as a <http://schema.org/DefinedTerm>. I guess that this could be a use case for other people as well, at least it is quite common to have controlled vocabularies for this property.

An exemplary entry would look like this:
```
"learningResourceType": {
    "type": "DefinedTerm",
    "name": "Experiment",
    "url": "http://w3id.org/openeduhub/vocabs/learningResourceType/experiment",
    "inDefinedTermSet": "http://w3id.org/openeduhub/vocabs/learningResourceType/"
    },
```

Thank you and looking forward to discussion!

**Comment by philbarker** [comment]
Looks good to me. I would make a similar change to `educationalUse`

**Comment by stuartasutton** [comment]
+1 to adding `DefinedTerm` as an expected type to both `learningResourceType` and `educationalUse`.

**Comment by jmarks** [comment]
I agree enabling a crosswalk between defined terms is a very good idea.
This brings-up one of the original ideas related to the AlignmentObject, to
be able to create more clarity in such a crosswalk between competency
frameworks. SKOS , ASN and CASE all allow and support types of correlations
and alignments between nodes of competency frameworks or any ontology. e.g.
exactMatchf, relatedTo, isPartOf, and even Broadens and Narrows. This seems
the correct way to deal with this important use case.

https://schema.org/AlignmentObject

**Comment by RichardWallis** [comment]
Fixed in V11.0 release.
